website/sidebars.js

```diff
@@ -822,6 +830,13 @@ const sidebarSettings = {
             "reference/resource-configs/where",
           ],
         },
+        {
+          type: "category",
+          label: "For unit tests",
+          items: [
+            "reference/resource-properties/unit-tests",
+          ],
+        },
         {
           type: "category",
```

Comment:
"Testing documentation" link on this page is now broken https://docs-getdbt-com-git-unit-testing-dbt-labs.vercel.app/reference/commands/test

![Screenshot 2024-02-14 at 11 00 29 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/53586774/f7fe9ae8-dc06-4eec-8201-9c27949c2ce0)

Reply:
Good catch! Fixed it on the page